portfolio/src/main/webapp/calendar.js

```diff
@@ -79,69 +80,69 @@ function createNewCalendarEventUserInput() {
     document.getElementById('new-event-name').value = EVENT_DEFAULT_NAME;
   }
 }
-
+ 
 /**
  * Creates a Date object based on a string that represents a time in HH:MM
  * format. This function assumes that the date is the current date when the
  * function is called.
  */
 function getTimeObject(timeString) {
```

Comment:
Consider clarifying in the method comment what timeString looks like. Maybe provide an example.

Reply:
Updated.